src/scss/_statusbar.scss

```diff
@@ -81,6 +81,10 @@
   height: 100%;
 }
 
+.uppy-StatusBar-contentPadding {
+  margin-right: 0.3em;
```

Comment:
I would use `1ch` in this case since it's horizontal text relative sizing (`ch` is relative to the character width, `em` to the character height).

Reply:
Cool, I didn't know about `ch`! Will update, thanks :)